roles/ssh_hardening/tasks/hardening.yml

```diff
@@ -101,8 +101,8 @@
   when: ssh_server_hardening | bool
 
 - name: Remove all small primes
-  ansible.builtin.shell: awk '$5 >= {{ sshd_moduli_minimum }}' {{ sshd_moduli_file }} > {{ sshd_moduli_file }}.new ;
-         [ -r {{ sshd_moduli_file }}.new -a -s {{ sshd_moduli_file }}.new ] && mv {{ sshd_moduli_file }}.new {{ sshd_moduli_file }} || true
+  ansible.builtin.shell: awk '$5 >= {{ sshd_moduli_minimum }}' {{ sshd_moduli_file }} > {{ sshd_moduli_file }}.new ; [ -r {{ sshd_moduli_file }}.new -a -s {{ sshd_moduli_file
```

Comment:
this line wrap look weird

Reply:
It does but it works and it is what ansible-lint proposes (and automatically fixes).

Reply:
good. Still this is the only change that does feel strange to me. I really dislike this wrap :D